Storage/ThirdParty/SwiftData.swift

```diff
@@ -812,8 +804,7 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
 
         // Attempt to make a backup as long as the database file still exists.
         if files.exists(baseFilename) {
-            log.warning("Couldn't create or update schema \(self.schema.name). Attempting to move \(baseFilename) to another location.")
-            SentryIntegration.shared.sendWithStacktrace(message: "Couldn't create or update schema \(self.schema.name). Attempting to move \(baseFilename) to another location.", tag: "SwiftData", severity: .warning)
+            Sentry.shared.sendWithStacktrace(message: "Couldn't create or update schema. Attempted to move db to another location.", tag: SentryTag.swiftData, severity: .warning, description: "\(self.schema.name). Attempting to move \(baseFilename)")
```

Comment:
Change to: `description: "Attempting to move '\(baseFilename)' for schema '\(self.schema.name)'"`